### DISCUSSION TITLE
How to change page for newtab

### DISCUSSION THREAD

**Discussion by TheArctesian** [qutebrowser/qutebrowser#7348] [Q&A] [answered]
I have a very simple [website](http://172.104.177.135:3333/) running on a linode instance that I use for my new tab on firefox.  I use 
```
c.url.start_pages = '172.104.177.135:3333/'
```
which works for on start, I don't know the command for changing new tab behaviour and would love help.

**Comment by The-Compiler** [comment, marked as answer]
You mean if you open a new blank tab (`:open -t`, rather than using `O` and entering a URL)? That'd be `url.default_page`.

**Reply by TheArctesian** [reply]
Fixed it thanks 
for others looking at this just made this change to my config.py
```py
c.url.default_page = homePage
```
where home page is the adr
